Add an internal onboarding guideline

BrianBohe opened the pull request:
How to get privileges to run /azp commands on your team's repos.
How to join Microsoft and .Net foundation organizations.

BrianBohe committed:
Add an internal onboarding guideline

How to get privileges to run /azp commands on your
team's repos.
How to join Microsoft and .Net foundation organizations.

BrianBohe commented:
@jaredpar

jkotas commented:
https://github.com/dotnet/core/blob/master/Documentation/microsoft-team.md is a very similar onboarding doc. Should we just link to the shared one?

jkotas commented on docs/workflow/intern-onboarding.md:
```diff
@@ -0,0 +1,20 @@
+    + Go to https://opensourcehub.microsoft.com/link/ and https://repos.opensource.microsoft.com/ to link.
+    + Go to https://repos.opensource.microsoft.com/Microsoft to join Microsoft org.
+    + Go to  https://repos.opensource.microsoft.com/DotNet to join DotNet org.
+    + Join this group as well https://repos.opensource.microsoft.com/dotnet/teams/mlnet if you want to do PR via codeflow.
```
Nit: What does ML.NET has to do with codeflow?

BrianBohe replied:
I merged the peaces of guidelines I got by interacting with other members for any new contributors (it took me a week to get /azp command privileges). Giving your previous comment, I think this file should only link that document and tells one to ask for the /azp commands privileges.

BrianBohe committed:
Update guideline

Remove repeted guidelines and point to the
existing one in dotnet/core Documentation.

BrianBohe committed:
Move azp priviliges guideline to pr-guide.md

danmosemsft commented:
There was a similar PR not long ago and the feedback was that we don't generally include docs that are only  relevant to Microsoft onboarding and not necessarily specific to this repo either. Feels like they should go on a one note or SharePoint or something else internal.

BrianBohe commented:
> There was a similar PR not long ago and the feedback was that we don't generally include docs that are only relevant to Microsoft onboarding and not necessarily specific to this repo either. Feels like they should go on a one note or SharePoint or something else internal.

I agree with you and we actually have one.
